CONTRIBUTING.rst

```diff
@@ -177,30 +175,40 @@ dependencies via:
 If modifying dependencies in ``mlflow/server/js/package.json``, run ``npm update`` within
 ``mlflow/server/js`` to install the updated dependencies.
 
+Launching the Development UI
```

Comment:
Do you think we need to have separate headings for Launching the Dev UI and Running the JS Dev Server. Can they be merged?

Reply:
The original document was written by @smurching, and when we restructured it we placed it as such. I'll leave that up to @smurching if it makes sense to merge them.